app/models/mentor_material.rb

```diff
@@ -30,3 +30,6 @@ def mentor_material_parts_in_order
     elements_in_order(elements: preloaded_parts, get_previous_element: :previous_mentor_material_part)
   end
 end
+
+MentorMaterialPart.where(mentor_materials: MentorMaterial.where(course_lesson: nil)).delete_all
+MentorMaterial.where(course_lesson: nil).delete_all
```

Comment:
:eyes:

Reply:
Well spotted, I should read my own PRs too 😂